constraint/config/samples/templates_v1alpha1_mutationtemplate.yaml

```diff
@@ -0,0 +1,9 @@
+apiVersion: templates.gatekeeper.sh/v1alpha1
+kind: MutationTemplate
+metadata:
+  labels:
+    controller-tools.k8s.io: "1.0"
+  name: mutationtemplate-sample
+spec:
+  # Add fields here
```

Comment:
Does this part need to be fixed?

Reply:
Do you think we should add more fields? It's currently the exact same as the constrainttemplate-sample

Reply:
In that case, no :)